src/main/java/shop/mtcoding/rodongin/controller/ResumeController.java

```diff
@@ -6,6 +6,10 @@
 @Controller
 public class ResumeController {
     
+    @GetMapping("/resume/resume")
```

Comment:
resume/resume -> resume
```suggestion
@GetMapping("/resume/1")
```